lesson_19/edmond_sabou/home1_les19.py

```diff
@@ -0,0 +1,37 @@
+
+from contextlib import contextmanager
+
+
+data = {
+    "Name": "Ubuntu",
+    "Version": "20.04",
+    "Install": "apt",
+    "Owner": "Canonical",
+    "Kernel": "5.4.0",
+    "Shell": "bash 5.0.16"
+}
+
+
+@contextmanager
+def just_some_exceptions(filename, method):
+    print("__enter__")
+    file = open(filename, method)
```

Comment:
the implementation is correct, but you don't need a file. The context manager is supposed to only catch exceptions not deal with files